server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/categories/CassandraAndElasticSearchOnly.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.james.jmap.categories;
+
+/**
+ * Category marker for Junit 4 conditional execution
+ * Marks tests that should be run only on Cassandra & ElasticSearch implementation
+ */
+public interface CassandraAndElasticSearchOnly {
```

Comment:
The name is bad as it lets me thinks the test only run on top of the Cassandra backend.

`CassandraElasticSearchTest` seems better to me.

Reply:
WDYT about `CassandraAndElasticSearchCategory`